autogpt_platform/backend/backend/data/notifications.py

```diff
@@ -409,44 +409,40 @@ async def create_or_add_to_user_notification_batch(
         )
 
         if not existing_batch:
-            async with transaction() as tx:
```

Comment:
@ntindle we can create a single statement query instead of starting transaction of two queries

Reply:
Yeah makes sense